acorn/src/statement.js

```diff
@@ -134,6 +134,7 @@ pp.parseStatement = function(context, topLevel, exports) {
       if (!this.inModule)
         this.raise(this.start, "'import' and 'export' may appear only with 'sourceType: module'")
     }
+    this.next()
```

Comment:
This is likely to break other plugins if they hook into `parseImport`/`parseExport`.

Reply:
I think so. MY BAD